src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/outertypenumber/Example2.java

```diff
@@ -0,0 +1,19 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="OuterTypeNumber">
+      <property name="max" value="2"/>
+    </module>
+  </module>
+</module>
+*/
+
+// xdoc section -- start
+package com.puppycrawl.tools.checkstyle.checks.sizes.outertypenumber;
+
+public class Example2 {
+}
+
+enum Outertype1 {
```

Comment:
There should be less violations after we increase max.
But we still have no violations.

Lets update example1 also to have some violation

Reply:
Example2 to have violations? or example1?

Reply:
Example1 can have 3 inner types. Has violation

Example2 has no violation as we put 3 as allowed max

Reply:
Please push your changes to let me see all code and error more clearly

Reply:
done